### PR TITLE
Update fluent.js

### PR DESCRIPTION
**Pull request by Be1zebub**
solved Uncaught TypeError: Cannot read properties of undefiend (reading 'Translations') at fluent.js:77

### CODE REVIEW

**Comment by Be1zebub** [comment]
also sidebar icons fixed
before:
before
after:
after

issue: #648

**Comment by Be1zebub** [comment]
Now it looks almost divine.

If we solve the problem with the nav buttons in header, it will be perfect.

File: Fluent/color.ini [hunk 1]
```diff
@@ -53,3 +57,30 @@ accent             = 00BEFD
 
 layer-shadow       = 1d1d1d
 contour            = 303030
+
+[comfy]
```
Comment (williamckha) [inline Fluent/color.ini:61]: Add a preview of this colour scheme in [THEMES.md](/morpheusthewhite/spicetify-themes/blob/master/THEMES.md)

**Review by williamckha — CHANGES_REQUESTED** [review]
Thanks for implementing the fixes. Sorry I haven't been maintaining this theme.
Unfortunately this solution of manually replacing the class names is temporary and the same issues will likely reoccur once Spotify updates. I was hoping that an updated css map in a new release of spicetify would resolve the issues, but spicetify hasn't released an update in a while.

**Comment by CharlieS1103** [comment]
@williamckha I attempted to fix the css map two weeks ago, however the changes didn't appear to change anything.

**Comment by williamckha** [comment]
> @williamckha I attempted to fix the css map two weeks ago, however the changes didn't appear to change anything.

Thanks for letting me know. It's most likely that class names have been simply changed/removed over time from the mappings

**Comment by CharlieS1103** [comment]
@williamckha If you look at https://github.com/khanhas/spicetify-cli/commit/0689d4eac70887b75528158c2078b03eb019630e, you'll see that I attempted to map them to the same icon class to fix this, however the change didn't seem to affect anything for whatever reason

**Comment by Be1zebub** [comment]
top bar back&forward buttons is also fixed, i used `:nth-child` to make it work for any spotify ver.
the only thing I can't fix is the play button above the playlist's thumb.

**Review by williamckha — APPROVED** [review]

**Pull request merged**